cmd/tesla-control/commands.go

```diff
@@ -453,6 +453,31 @@ var commands = map[string]*Command{
 			return car.ChargeStop(ctx)
 		},
 	},
+	"charging-schedule": &Command{
+		help:             "Schedule charging to xx minutes after midnight and enable daily scheduling",
```

Comment:
The convention is to use the Argument name in the help string, so "...to MINS minutes" instead of "...to xx minutes".

Reply:
ok, fixed now. Thanks!